packages/lesswrong/components/comments/CommentsList.jsx

```diff
@@ -70,7 +70,7 @@ class CommentsList extends Component {
           <div>
             {comments.map(comment =>
               <Components.CommentsNode
-                startThreadTruncated={startThreadTruncated || totalComments >= 50}
+                startThreadTruncated={startThreadTruncated || totalComments >= 100}
```

Comment:
Why?

Reply:
it's the right number (mostly just due to experimentation and noticing that 50 comments was too early to do the thing)